### ISSUE TITLE
transport/server: info log entry about exception on client disconnection

### ISSUE THREAD

**Issue by zimnx** [scylladb/scylla#6775]
*Installation details*
Scylla version (or git commit hash): 4.1

Code suggests that broken pipes shouldn't be logged, but they are.

```
TRACE 2020-07-06 12:37:50,097 [shard 1] cql_server - Advertising new connection from CQL client 192.168.100.1:35554                                                                                                                                                                                                                                                                                                              
TRACE 2020-07-06 12:37:50,097 [shard 1] cql_server - Advertising disconnection of CQL client 192.168.100.1:35554                                                                                                                                                                                                                                                                                                                 
INFO  2020-07-06 12:37:50,098 [shard 1] cql_server - exception while processing connection: std::system_error (error system:32, sendmsg: Broken pipe) 
```
https://github.com/scylladb/scylla/blob/master/transport/server.cc#L270-L282

Reproducer: 
1. Spawn cluster
2. Add cluster to Scylla Manager
3. Every 15s when healthcheck is triggered, log entry appears.

SM healthcheck creates new connection, queries system table and disconnects.

**Comment by avikivity** [comment]
Fixed in 4.1.1.

**Comment by avikivity** [comment]
(b8f7fb35e149ee6eb1bdbbc5cf9b4aa55772ac45)
